### DISCUSSION TITLE
How can I add a background image?

### DISCUSSION THREAD

**Discussion by Aaroneisele55** [dbuezas/lovelace-plotly-graph-card#400] [Q&A]
Hello,
I would like to add a background image to the plotly-graph-card.
How could I do this?
Could I even rotate and position it straight from the card UI?

Best regards
Aaron

**Comment by dbuezas** [comment]
See here: https://github.com/dbuezas/lovelace-plotly-graph-card/discussions/48

**Reply by Aaroneisele55** [reply]
I already saw that, but I don't quite understand how to use the results.
Could you please show me how ?
I am a real noob with HA Dashboards...
Best regards 
Aaron

**Reply by dbuezas** [reply]
Hi Aaron,
I don't use background images so all I have is that link.
Maybe @Photelegy can help you if he sees this :)
